notebooks/264-speculative-sampling/264-speculative-sampling.ipynb

```diff
@@ -0,0 +1,492 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #58.            target_past_kv = target_new_past_kv
</pre><p>You should update <code>target_past_kv</code> based on the number of accepted tokens.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>